book/06-github/sections/5-scripting.asc

```diff
@@ -171,24 +141,24 @@ hs_err_pid*
 ----
 
 
-==== Commenting on an Issue
+==== Commenter un problème
 
-However, if you want to do an action on the website such as comment on an Issue or Pull Request or if you want to view or interact with private content, you'll need to authenticate.
+Cependant, si vous voulez faire une action sur le site web comme commenter un problème ou une requête de tirage ou si vous voulez voir ou interagir avec du contenu privé, vous aurez besoin de vous authentifier.
 
-There are several ways to authenticate. You can use basic authentication with just your username and password, but generally it's a better idea to use a personal access token.
-You can generate this from the ``Applications'' tab of your settings page.
+Il y a plusieurs moyens de s'authentifier. Vous pouvez utiliser l'authentification basique avec seulement votre nom d'utilisateur et votre mot de passe, mais en général c'est mieux d'utiliser un jeton d'accès personnel.
+Vous pouvez en générer depuis l'onglet ``Applications'' de votre page de paramètres.
```

Comment:
Comment fait-on les cotes françaises ? Y a-t-il une façon particulière ou faut-il insérer les caractères '«' et '»' ?

Date: Fri, 13 Nov 2015 13:08:08 -0800
From: notifications@github.com
To: progit2-fr@noreply.github.com
CC: adr.ollier@hotmail.fr
Subject: Re: [progit2-fr] Chap6 (#53)

In book/06-github/sections/5-scripting.asc:

>  
> -There are several ways to authenticate. You can use basic authentication with just your username and password, but generally it's a better idea to use a personal access token.
> -You can generate this from the ``Applications'' tab of your settings page.
> +Il y a plusieurs moyens de s'authentifier. Vous pouvez utiliser l'authentification basique avec seulement votre nom d'utilisateur et votre mot de passe, mais en général c'est mieux d'utiliser un jeton d'accès personnel.
> +Vous pouvez en générer depuis l'onglet ``Applications'' de votre page de paramètres.

Non, le cotes représentées dans le texte d'origine permettent d'avoir des cotes anglaises correctes, comme “ceci”, alors qu'on veut de cotes française comme « ceci ».


—
Reply to this email directly or view it on GitHub.